.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -18,6 +18,31 @@ body:
         - label: I agree to follow this project's [Code of Conduct](https://github.com/golangci/golangci-lint?tab=coc-ov-file)
           required: true
 
+  - type: dropdown
+    id: install
+    attributes:
+      label: How did you install golangci-lint?
+      options:
+        - I don't know
```

Comment:
`Other` and `I don't know` are similar choices, we can merge it into `Other`.

Reply:
this is different: from my experience, some people don't read and use the first element.